conans/client/command.py

```diff
@@ -91,7 +92,11 @@ def _fill_text(self, text, width, indent):
 _KEEP_SOURCE_HELP = ("Do not remove the source folder in local cache, even if the recipe changed. "
                      "Use this for testing purposes only")
 _PATTERN_OR_REFERENCE_HELP = ("Pattern or package recipe reference, e.g., '%s', "
-                              "'%s'" % (_REFERENCE_EXAMPLE, _PATTERN_EXAMPLE))
+                              "'%s'" % (_PATTERN_EXAMPLE, _REFERENCE_EXAMPLE))
+_PATTERN_REF_OR_PREF_HELP = ("Pattern, recipe reference or package reference e.g., '%s', "
+                              "'%s', '%s'" % (_PATTERN_EXAMPLE, _REFERENCE_EXAMPLE, _PREF_EXAMPLE))
+_PREF_OR_PREF_HELP = ("Recipe reference or package reference e.g., '%s', "
```

Comment:
```suggestion
_REF_OR_PREF_HELP = ("Recipe reference or package reference e.g., '%s', "
```

or `_REFERENCE_OR_PREF_HELP`